nixos/modules/config/users-groups.nix

```diff
@@ -352,7 +357,8 @@ let
           promptly.
 
           Note that the {option}`password` option will override this
-          option if both are set.
+          option if both are set. One exception to this rule is if
+          `systemd.sysusers.enable = true;`.
```

Comment:
Hmm after reading through the code it seems to me as if all non-initial password options would override this.

Reply:
I might not understand what you mean, but I think the test on line 35 of nixos/tests/password-option-override-ordering.nix that I added shows that initialPassword actually overrides hashedPassword (a non-initial password option), which conflicts with what you are saying?

Reply:
Only for `mutableUsers = false`; though, see https://github.com/NixOS/nixpkgs/blob/400e3ecf59205c6197e093e571e6518a669c4d55/nixos/modules/config/users-groups.nix#L417-L421